### DISCUSSION TITLE
Workflow Job status updated but now it stuck in the in-progress mode

### DISCUSSION THREAD

**Discussion by ShyamModha** [community/community#125013] [Actions] [answered]
I triggered my workflows around 30 minutes ago and all were in the queued mode only. When i attempt to cancel, it fails. After 25 mins one of workflow's status changed to in-progress

**Comment by eullercdr** [comment]
The same happens here.

**Comment by CharlesParent** [comment]
Same for me as well !
Some jobs are also still displayed as In Progress whereas all tasks in it are completed

**Comment by ShyamModha** [comment]
Just now some of the jobs are completed its process successfully, but new ones are still getting queued.

**Comment by jgresc** [comment]
Same here.
Sometimes the workflows "jumps", but mostly it's in queue

**Comment by CharlesParent** [comment]
Not sure, but it looks like hanging worflows are stuck at the following step:

`Cleaning up orphan processes`

This is an automatic task that is launched at the end of each workflow

**Comment by CHIRAGAGARWAL123** [comment]
GitHub workflow stuck at 

Waiting for a runner to pick up this job...
Job is about to start running on the runner

is there any update from GitHub side?

**Comment by davevad93** [comment]
Hi @ShyamModha ,

There is an issue occurring with `Actions` as indicated in [GitHub Status](https://www.githubstatus.com/incidents/nj47vccwm2zj), for now you can just wait for it to be fixed.

**Comment by ricardomar** [comment]
same here.
stuck on "Cleaning up orphan processes"

**Comment by MadhuPole** [comment]
Hi Everyone
Please check the status here.
https://www.githubstatus.com/

**Comment by grandmas-favorite** [comment, marked as answer]
Hi everyone!

[The incident has been updated](https://www.githubstatus.com/incidents/nj47vccwm2zj) and Actions is now operating normally.
